core/domain/skill_domain.py

```diff
@@ -483,9 +483,13 @@ def require_valid_description(cls, description):
         if description == '':
             raise utils.ValidationError('Description field should not be empty')
 
-    def validate(self):
+    def validate(self, new_skill=False):
```

Comment:
I think all skills (that are saved in the datastore) should have the full set of rubrics from the get-go.

This flag seems to be used only in create_new_skill which has a committer_id, so by that point there should definitely be the correct 3 rubrics.

Reply:
Done